itests/src/test/java/org/apache/unomi/itests/migration/Migrate16xTo220IT.java

```diff
@@ -84,8 +84,16 @@ public void checkMigratedData() throws Exception {
         checkViewEventRestructured();
         checkEventTypesNotPersistedAnymore();
         checkForMappingUpdates();
+        checkNewIndexesExists();
     }
 
+    /**
+     * Checks if at least the new index for events and sessions exists.
+     */
+    private void checkNewIndexesExists() throws IOException {
+        Assert.assertTrue(MigrationUtils.indexExists(httpClient, "http://localhost:9400", "context-event-000001"));
+        Assert.assertTrue(MigrationUtils.indexExists(httpClient, "http://localhost:9400", "context-session-000001"));
+    }
```

Comment:
Would be great to check that the number of sessions/events is same as before the migration.
We could do a count query on context-session-* and context-event-* before starting the migration and compare after the migration
Also we could check that old indices do not exists anymore: context-session-date-* and same for events.